utils/githooks/pre-commit.d/20-flake.sh

```diff
@@ -21,6 +21,11 @@ if ! command -v flake8 > /dev/null 2>&1; then
     exit 0
 fi
 
+if [ ! -f .flake8 ]; then
+    echo "  No .flake8, skipping flake checks"
+    exit 0
+fi
+
```

Comment:
Is this change intentional?

Reply:
It is.  I don't recall the specific issue though.  I am trying to make the githooks more portable to more repositories and must have run into some issue with `.flake8` being missing.

Reply:
Nice idea.